backend/service/k8s/deployment_test.go

```diff
@@ -245,3 +246,67 @@ func TestProtoForDeploymentClusterName(t *testing.T) {
 		})
 	}
 }
+
+func TestProtoForDeploymentStatus(t *testing.T) {
+	t.Parallel()
+
+	var deploymentTestCases = []struct {
+		id                      string
+		expectedReplicas        uint32
+		expectedUpdatedReplicas uint32
```

Comment:
don't see the fields expectedReplicas & expectedUpdatedReplicas being used in the test cases below